flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/declarative/RestartingTest.java

```diff
@@ -209,6 +210,7 @@ public void close() throws Exception {
                     NoOpExecutionDeploymentListener.get(),
                     (execution, newState) -> {},
                     0L);
+            setJsonPlan("");
```

Comment:
.

Reply:
This will be removed again once the ExecutionGraph interface PR has been merged. If I don't set the JSON plan, creating an archived execution graph will fail.